src/main/java/org/broadinstitute/hellbender/tools/variantdb/SampleList.java

```diff
@@ -81,8 +82,9 @@ private TableResult querySampleTable(String fqSampleTableName, String whereClaus
                 "SELECT " + SchemaUtils.SAMPLE_ID_FIELD_NAME + ", " + SchemaUtils.SAMPLE_NAME_FIELD_NAME +
                 " FROM `" + fqSampleTableName + "`" + whereClause;
 
-        // Execute the query:        
-        final TableResult result = BigQueryUtils.executeQuery(BigQueryUtils.getBigQueryEndPoint(executionProjectId) , sampleListQueryString, false);
+
+        // Execute the query:
+        final TableResult result = BigQueryUtils.executeQuery(sampleListQueryString, null);
```

Comment:
Change this back to the invocation that supplied the projectId, but also includes your `null` for the labels.  e.g.

```
final TableResult result = BigQueryUtils.executeQuery(BigQueryUtils.getBigQueryEndPoint(executionProjectId) , sampleListQueryString, false, null);
```